base/file.jl

```diff
@@ -401,11 +401,11 @@ struct uv_dirent_t
 end
 
 """
-    readdir(dir::AbstractString=".") -> Vector{String}
+    readdir(dir::AbstractString="."; sort::Bool=true) -> Vector{String}
 
-Returns the files and directories in the directory `dir` (or the current working directory if not given).
+Returns the files and directories in the directory `dir` (or the current working directory if not given). If `sort=false` the resulting list won't necessarily be sorted.
```

Comment:
Can you wrap this? We're not very good about consistently wrapping text but this is getting a bit long.